server.go

```diff
@@ -252,6 +276,27 @@ func (s *Server) HandleMetricPacket(packet []byte) {
 	}
 }
 
+func (s *Server) HandleTracePacket(packet []byte) {
+	// Unlike metrics, protobuf shouldn't have an issue with 0-length packets
+	if len(packet) == 0 {
+		log.Error("received zero-length trace packet")
+	}
+
+	// Technically this could be anything, but we're only consuming trace spans
+	// for now.
+	newSample := &ssf.SSFSample{}
+	err := proto.Unmarshal(packet, newSample)
+	if err != nil {
+		log.Fatal("Trace unmarshaling error: ", err)
+	}
+
+	s.statsd.Count("veneur.traces.handled", 1, []string{}, 1.0)
```

Comment:
so - how much volume of traces are we expecting to get? my past experience is that reporting metrics from within a udp hot loop is _very_ expensive because it incurs additional io. can we report this metric from the other side instead? (ie instead of incrementing a counter each time we get a trace, report a total counter at flush time)